lib/collector/remote-method.js

```diff
@@ -142,6 +147,9 @@ RemoteMethod.prototype._post = function _post(data, nrHeaders, callback) {
     makeRequest()
   }
 
+  /**
```

Comment:
does this remove warnings, if so that kinda stinks

Reply:
This specifically has been removed but also these "empty comment blocks" have been made non-empty in all our public-facing docs.